deps/parser/parser.jl

```diff
@@ -163,54 +162,96 @@ function parse_definition(method_dict::Dict, app_name::String)
         else
             call_args = join(argument_list[1:number_arguments],",")
         end
-        return_string = return_string * function_string_creator(julia_name,polymake_name,app_name,julia_args,call_args,parameter_string)
+        if is_method
+            method_string = method_string * julia_method_string(julia_name,polymake_name,app_name,julia_args,call_args,parameter_string)
+        else
+            function_string = function_string * julia_function_string(julia_name,polymake_name,app_name,julia_args,call_args,parameter_string)
+        end
         if has_option_set
             julia_args = julia_args * "," * option_set_argument
             call_args = call_args * "," * option_set_parameter
-            return_string = return_string * function_string_creator(julia_name,polymake_name,app_name,julia_args,call_args,parameter_string)
+            if is_method
+                method_string = method_string * julia_method_string(julia_name,polymake_name,app_name,julia_args,call_args,parameter_string)
+            else
+                function_string = function_string * julia_function_string(julia_name,polymake_name,app_name,julia_args,call_args,parameter_string)
+            end
         end
     end
-    return return_string
+    return function_string,method_string
 end
 
-function parse_app_definitions(filename::String,outputfileposition::String,include_file::String)
+module_string_includes = """
+
+import ..pm_Integer, ..pm_Rational, ..pm_Matrix, ..pm_Vector, ..pm_Set,
```

Comment:
Can you explain why we need the imports here? Also I think this is not a great idea, since we need to keep all these exports and imports in sync and this will become a mess...